game.test.js

```diff
@@ -0,0 +1,60 @@
+// const { it } = require('jest-circus');
+const Game = require('./game');
+
+let game;
+
+beforeEach(() => {
+  game = new Game;
+});
```

Comment:
try doing a test for all spares perhaps, which scores 190 (roll of 9 followed by 1 for every frame, with the third roll in the final frame being 9)